docs/api_docs/chat.md

```diff
@@ -0,0 +1,46 @@
+# Chat API
+
+#### `chat/{user_id}/messages`
+`GET`
+- Returns all messages sent to/from the requested user
+- Requires the requested user to be logged in
```

Comment:
Should we even include `{user_id}` if the user needs to be logged in?

Reply:
Maybe not, but it could be useful to keep as an extra layer of validation so that we're never accidentally displaying messages to the wrong user.

Reply:
Is there a known situation where that could happen?

I'm fine with either approach *(and it's not a big deal given our scope)* but my main concern is consistency. If we're keeping `user_id` here despite it requiring login, then I'd argue other endpoints that require the user to be logged in should act the same, and vice versa.

Reply:
Not a known situation, just hypothetically speaking.

I'll remove the `user_id` and if a situation comes up where it's a problem we can address it then.